### ISSUE TITLE
[AXE-CORE]: Find Forms - Description lists must have valid markup

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#23851]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Details

The VA.gov /find-forms/FORM_NAME pages have an axe-core violation in production related to invalid description list markup. These will start breaking builds when work from https://github.com/department-of-veterans-affairs/va.gov-team/issues/21738 is merged in. Screenshot and snippet attached below.

## Acceptance Criteria

- [ ] HTML validates a checker like W3C HTML5 validator
- [ ] 0 axe-core violations thrown
- [ ] CONSIDER using the Liquid unit tester ( https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/testing/liquid-templates/README.md )

## Environment

* https://www.va.gov/find-forms/about-form-26-6808/ (this is a representative sample)

## Steps to Recreate

1. Enter https://www.va.gov/find-forms/about-form-26-6808/ after opting out of production analytics
2. Run the [axe-core browser plugin](https://deque.com/axe) for Chrome or Firefox
3. Verify the description list violation

## Proposed Solution (if known)

The issue appears to be on [line 28 of the va_form.drupal.liquid](https://github.com/department-of-veterans-affairs/vets-website/blob/2ac31d59c12f863def55aad7aeebc274073bc499/src/site/layouts/va_form.drupal.liquid#L28) template:

```diff
{% else %}
- <dfn class="vads-u-visibility--screen-reader">Form name:</dfn>
  <dt class="va-introtext">
+   <dfn class="vads-u-visibility--screen-reader">Form name:</dfn>
{% endif %}
```

## WCAG or Vendor Guidance (optional)

* https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-parses.html

## Screenshots or Trace Logs

<img width="1071" alt="Screen Shot 2021-04-27 at 10 37 22 AM" src="https://user-images.githubusercontent.com/934879/116272510-92f4d400-a746-11eb-98c6-c8141064c60b.png">

**Comment by MarciMcGuireGCIO** [comment]
@zacharymorel since this is a defect-1 please have a look as soon as you get to a good stopping point with your current task.  Thanks!

**Comment by zacharymorel** [comment]
She thing @MarciMcGuireGCIO

**Comment by zacharymorel** [comment]
@1Copenut 

This is on Staging now if you'd like to review when you get the chance.

**Comment by 1Copenut** [comment]
@zacharymorel & @MarciMcGuireGCIO I retested the linked page in prod, and the axe violation is gone. Since it's a template, I have high confidence all of these have gone away. I'll move this ticket to close and pull your code into my Nightwatch update branch. Thank you for prioritizing this issue!

**Comment by MarciMcGuireGCIO** [comment]
Thank you, @1Copenut! Great job, @zacharymorel!
